kafka/modules/kafka/base/module.yaml

```diff
@@ -11,7 +11,7 @@ envs:
   - name: "S2I_HOME"
     value: "/opt/kafka/s2i"
   - name: "KAFKA_EXPORTER_HOME"
-    value: "/opt/kafka-exporter"
+    value: "/usr/bin"
```

Comment:
Since we are going to install the Kafka Exporter via RPM we can safelty remove this env var

Reply:
Isn't it still needed for https://github.com/strimzi/strimzi-kafka-operator/blob/c0620b286b2cbe5e529e6f8b6374327681915fd0/docker-images/kafka/exporter-scripts/kafka_exporter_run.sh#L40 ?

Reply:
Ah yes you are right!, forget what I said, this is good to go!